### PR TITLE
Bump to scala 2.13.10 / chisel 3.5.5

### PR DESCRIPTION
**Pull request by jerryz123**
**Related issue**: 

**Type of change** other enhancement

**Impact**: API modification

**Development Phase**:  implementation

**Release Notes**

### CODE REVIEW

File: src/main/scala/util/HeterogeneousBag.scala [hunk 1]
```diff
@@ -10,6 +10,7 @@ final case class HeterogeneousBag[T <: Data](elts: Seq[T]) extends Record with c
   def apply(x: Int) = elts(x)
   def length = elts.length
 
+  override def className: String = stringPrefix
```
Comment (davidbiancolin) [inline src/main/scala/util/HeterogeneousBag.scala:13]:
```suggestion
  override def className: String = super.className
```

File: src/main/scala/subsystem/HasTiles.scala [hunk 2]
```diff
@@ -355,21 +355,25 @@ trait CanAttachTile {
...
-        tileClockGroup
+        domain.clockNode := tileClockGroup
       }
     })
 
     domain {
       domain.tile_reset_domain.clockNode := crossingParams.resetCrossingType.injectClockNode := domain.clockNode
-    } := clockSource
+    }
+    //domain.clockNode := clockSource
```
Comment (davidbiancolin) [inline src/main/scala/subsystem/HasTiles.scala:376]: Are you just waiting to test before removing this?

**Review by davidbiancolin — APPROVED** [review]
These look sane to me.

File: build.sbt [hunk 3]
```diff
@@ -10,14 +10,13 @@ val chiselVersion = "3.5.4"
 lazy val commonSettings = Seq(
   organization := "edu.berkeley.cs",
   version      := "1.2-SNAPSHOT",
-  scalaVersion := "2.12.15",
+  scalaVersion := "2.13.8",
```
Comment (jackkoenig) [inline build.sbt:13]: Any reason not to just go for 2.13.10?

Comment (jerryz123) [reply build.sbt:13]: No reason. I can bump to that

**Comment by sequencer** [comment]
I personally suggest directly remove the bucket 9, and readd in the future, the ScalaFix in 213 is more strict then 212, and we should fix that in another PR.

**Pull request merged**